src/main/java/com/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,63 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
```

Comment:
Коректно називати таблиці в множині, якщо я правильно запам'ятав :)
```suggestion
@Table(name = "movie_sessions")
```